source/beamline/BioXAS/BioXASSideMonochromator.cpp

```diff
@@ -45,7 +45,7 @@ BioXASSideMonochromator::BioXASSideMonochromator(QObject *parent) :
 	region_->setPaddleStatusControl(paddleStatus_);
 	region_->setKeyStatusControl(keyStatus_);
 	region_->setBrakeStatusControl(brakeStatus_);
-	region_->setBraggControl(bragg_);
+	region_->setBraggControl(stepBragg_);
```

Comment:
This is one of the examples where I'm confused.  All other changes in this .cpp file is switched to `encoderBragg_` except this one.